.github/workflows/multi_nim_devel.yml

```diff
@@ -0,0 +1,79 @@
+name: Daily - Nim Devel
```

Comment:
This file is basically a duplicate of the previous one?

Reply:
Please have a look also at the first commit. Files may look similar, but "Daily" is looking at present problems and the "Daily - Nim Devel" is looking into future potential problems. This separation would reduce our need for immediate attention. This would give us some mental space to schedule new features development rather than chasing potential issues. I am not the only person in the team, so also depends on you guys which way would you like to go ?

Reply:
The comment was more related to the technical aspect of the proposed change.

Reply:
What would you like to change at the moment ?

Reply:
Avoid the whole duplication of the file.

Reply:
Please have a look at latest commit.